static/app/components/events/eventReplay/index.tsx

```diff
@@ -28,7 +30,7 @@ export default function EventReplay({event, group, projectSlug}: Props) {
     return <ReplayClipSection event={event} replayId={replayId} group={group} />;
   }
 
-  if (canShowUpsell) {
+  if (canShowUpsell && !hasSentOneReplay) {
```

Comment:
what does canShowUpsell check?

Reply:
https://github.com/getsentry/sentry/blob/0887140eb635c990b2b64f3ec2218feee3669fb9/static/app/utils/event/useEventCanShowReplayUpsell.tsx#L53-L55 

it checks
1) the platform is supported by replay
2) either the org has not sent any replays or the platform is part of the replay backend platforms

Reply:
oh why doesn't the check for the org not sending any replays work in this case?

Reply:
bc it's an or (`||`) so any backend platform (e.g. the platforms that we show the banner for) will automatically be true

Reply:
oh is there a reason why we can't remove the backend platform condition?

Reply:
the check targets whether the entire org has sent replays, but we want to show the backend banner even if the org has sent replays bc that specific project might not have sent replays yet, so we always override it with the backend platform check